File: RabbitMQ/demoPy/Producer.py
import pika

def main():
    # conectar con RabbitMQ
    connection = pika.BlockingConnection(pika.ConnectionParameters('localhost'))
    channel = connection.channel()

    # Declarar la cola
    channel.queue_declare(queue='hello')

    # Publicar un mensaje
    channel.basic_publish(exchange='', routing_key='hello', body='Hello RabbitMQ!')

    print(" [x] Sent 'Hello RabbitMQ!'")

    # Cerrar la conexión
    connection.close()

if __name__ == '__main__':
    main()

File: RabbitMQ/demo/src/main/java/com/example/Producer.java
package com.example;

import com.rabbitmq.client.Channel;
import com.rabbitmq.client.Connection;
import com.rabbitmq.client.ConnectionFactory;

public class Producer {

    public static void main(String[] args) {

        ConnectionFactory factory = new ConnectionFactory();
        factory.setHost("localhost");

        try
        {
            Connection connection = factory.newConnection();
            Channel channel = connection.createChannel();
            channel.queueDeclare("hello", false, false, false, null);
            String message = "Hello, RabbitMQ!";
            channel.basicPublish("", "hello", null, message.getBytes());
            System.out.println(" [x] Sent '" + message + "'");
            channel.close();
            connection.close();

        }catch(Exception e)
        {
            e.printStackTrace();
        }
    }
    
}
